pilot/pkg/bootstrap/config_compare.go

```diff
@@ -46,6 +46,17 @@ func needsPush(prev config.Config, curr config.Config) bool {
 			return true
 		}
 	}
+	// If current metadata has "*istio.io" label/annotation, just push
```

Comment:
This fix is needed if the previous has istio annotation for example exportTo and in current that is removed, we should still push to reflect the current state.